### DISCUSSION TITLE
Moving Client-Folder

### DISCUSSION THREAD

**Discussion by wofo99** [fedorovvl/tso_client#283] [Q&A]
Hi. I moved my Client-folder to a different host. Starting the client there resulted in bad_settings.dat and nothing is started. Is settings.dat somehow machine-related?

**Comment by fedorovvl** [comment]
settings.dat encrypted using DPAPI
protected data is associated with the machine context and cant be unencrypted on another machine

**Comment by wofo99** [comment]
pity.. so I have to recreate it on the target host whenever I switch 
machines, correct?
Anyway, thanks for the clarifying answer, it solved the riddle 😉



------------------------------------------------------------------------
Gesendet mit der Telekom Mail App
<https://kommunikationsdienste.t-online.de/redirects/email_app_android_sendmail_footer>


--- Original-Nachricht ---
Von: Vladimir
Betreff: Re: [fedorovvl/tso_client] Moving Client-Folder (Discussion #283)
Datum: 19. Juni 2023, 23:02
An: fedorovvl/tso_client
Cc: wofo99, Author



settings.dat encrypted using DPAPI
protected data is associated with the machine context and cant be 
unencrypted on another machine

—
Reply to this email directly, view it on GitHub
<https://github.com/fedorovvl/tso_client/discussions/283#discussioncomment-6223560> 
, or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AF2UJ6XAE3IQEXY2Q6ZN3LTXMC455ANCNFSM6AAAAAAZMKSSP4> 
.
You are receiving this because you authored the thread.Message ID: 
***@***.***>

**Comment by fedorovvl** [comment]
it will be auto created on first login
u planning to change pc every day? =)

**Comment by wofo99** [comment]
the idea was to have the Client folder on a stick and so being able to 
carry it around and game on different locations, eg. while babysitting 😉



------------------------------------------------------------------------
Gesendet mit der Telekom Mail App
<https://kommunikationsdienste.t-online.de/redirects/email_app_android_sendmail_footer>


--- Original-Nachricht ---
Von: Vladimir
Betreff: Re: [fedorovvl/tso_client] Moving Client-Folder (Discussion #283)
Datum: 20. Juni 2023, 0:36
An: fedorovvl/tso_client
Cc: wofo99, Author



it will be auto created on first login
u planning to change pc every day? =)

—
Reply to this email directly, view it on GitHub
<https://github.com/fedorovvl/tso_client/discussions/283#discussioncomment-6224048> 
, or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AF2UJ6XCLQPLA4DNUOI3OJTXMDH7LANCNFSM6AAAAAAZMKSSP4> 
.
You are receiving this because you authored the thread.Message ID: 
***@***.***>
